packages/amplify_datastore_plugin_interface/lib/src/types/temporal/temporal_datetime.dart

```diff
@@ -0,0 +1,157 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import 'temporal.dart';
+
+/// Represents a valid extended ISO-8601 DateTime string.  The time zone offset is compulsory.
```

Comment:
>The time zone offset is compulsory.

Why is this a requirement? DartTime doesn't have this.

Reply:
https://docs.aws.amazon.com/appsync/latest/devguide/scalars.html#appsync-defined-scalars

> The time zone offset is compulsory for this scalar.